docs/CaRROT-Mapper/quickstart.md

```diff
@@ -102,34 +101,34 @@ The Tables list will now show both tables have filled `PERSON ID` and `EVENT DAT
 
 ### Generate Initial Mapping Rules
 Navigate to the Mapping Rules page associated to your Scan Report. 
-This can be either (a) from the Scan Reports page - press the `Rules` button - ; or (b) from any of the Tables, Fields or Values pages - press the `Go to Rules` button near the top of the page.
+This can be either **(a)** from the Scan Reports page - press the `Rules` button; or **(b)** from any of the Tables, Fields or Values pages - press the `Go to Rules` button near the top of the page.
 This page will contain an empty table with the contents `Nothing`. 
 
-But we have Concepts already mapped for some of our Fields and Values, so what is going on?
+Some of the Fields and Values already have associated Concepts based on the Dictionary file and reuse of previously seen mappings.
```

Comment:
```suggestion
This might be surprising, as some of the Fields and Values already have associated Concepts based on the Dictionary file and reuse of previously seen mappings.
```

Reply:
Will the user actually see the Fields and Values that are already filled? Could this paragraph in fact just create a little confusion?

Reply:
If they navigate to the Fields or Values pages (as they will have done earlier, line 82) they've seen that there are Concepts associated to some of the Fields and Values.

Reply:
Ultimately I'm trying to pre-empt complaints of "this is confusing" by recognising that yes it _can_ be confusing, and explicitly explaining the reasoning.